frontend/src/components/utils/ListHandler.tsx

```diff
@@ -76,13 +70,17 @@ const Wrapper = <T, I>(config: StaticConfig<T, I>): FC<Props<T, I>> => {
             {!props.detailPage ? (
               <DatabaseConsumer>
                 {(context) => (
-                  <div onClick={fetch}>
-                    {config.category === "Block" ? (
-                      <Update>{`${translate(
-                        "utils.ListHandler.last_block"
-                      ).toString()}#${context.latestBlockHeight}.`}</Update>
+                  <>
```

Comment:
Is it a visual issue? Can you please show screenshots of how does that look without adding conditional?

Reply:
Sometimes instead of number of block `undefined` is shown

Reply:
You're right, It's just a visual issue

Reply:
Should we fix it separately?
I'm all in atomic changes :)

Reply:
Do you want to remove it from here and fix it by you own?

Reply:
It's ok if you fix it, but it would be really nice to have it in a different PR (which should be called something like "undefined in list handler"), it would be easier to blame it later that way.

Reply:
Ok, I'll move it to another PR